docker/README.md

```diff
@@ -0,0 +1,128 @@
+# Dockerizing
+
+This video is a good resource for tips on how to optimize docker setup
+
+https://youtu.be/kG2vxYn547E?t=255 
+
+TLDR;
+
+[See slides here](https://speakerd.s3.amazonaws.com/presentations/55d8979b402d43f4a5a99cce56273579/containers.pdf)
+
+## Tip #1: Read and understand your base image
+
+There is an official [ruby image](https://hub.docker.com/_/ruby) on Docker Hub, but it comes with batteries included. 
+
+[Alpine](https://hub.docker.com/_/alpine) is a very slim linux that can be extended with ruby.
```

Comment:
I do not recommend using Alpine. It's definitely slimmer, but you'll end up spending a lot of time installing packages.

Reply:
I agree completely with you on that. I prefer to use the official ruby image.

If we learned how to use the Multi-stage docker feature then it would make sense start of with the official ruby version and then kick off an slim image.